splash/qtrender_lua.py

```diff
@@ -54,6 +54,16 @@ def decorator(meth):
     return decorator
 
 
+def attr_function(attr_name, gettr_method):
+    """ Decorator for marking methods that make attributes available to Lua """
+    def decorator(meth):
+        meth._is_attr_func = True
+        meth._attr_name = attr_name
+        meth._gettr_method = gettr_method
+        return meth
+    return decorator
```

Comment:
What about renaming this decorator to `@lua_property`? It is similar to `@property` decorator, but creates a property available to Lua code.